using_images/other_images/jenkins.adoc

```diff
@@ -75,10 +177,51 @@ The Jenkins password can be configured with the following environment variable:
 |Variable name |Description
 
 |`*JENKINS_PASSWORD*`
-|Password for the `*admin*` user.
+|Password for the `*admin*` user when using standard Jenkins authentication.  Not applicable when using {product-title} OAuth authentication.
```

Comment:
i can't comment on it in this PR but line 171 above should be reworded to:  "The Jenkins server can be configured with the following environment variables:"